helm/templates/deployment.yaml

```diff
@@ -36,7 +37,14 @@ spec:
           ports:
             - name: http
               containerPort: 9090
+          volumeMounts:
+            - name: keystore-script-volume
```

Comment:
Why is this file mounted?

Reply:
This is mounting the config map. So the scripts can be read by the init containers

Reply:
This has been removed